src/sagemaker/tensorflow/model.py

```diff
@@ -165,22 +158,26 @@ def prepare_container_def(self, instance_type, accelerator_type=None):
 
         return sagemaker.container_def(deploy_image, self.model_data, deploy_env)
 
-    def serving_image_uri(self, region_name, instance_type):
+    def serving_image_uri(self, region_name, instance_type, accelerator_type=None):
         """Create a URI for the serving image.
 
         Args:
             region_name (str): AWS region where the image is uploaded.
             instance_type (str): SageMaker instance type. Used to determine device type
                 (cpu/gpu/family-specific optimized).
+            accelerator_type (str): The Elastic Inference accelerator type to
+                deploy to the instance for loading and making inferences to the
+                model (default: None). For example, 'ml.eia1.medium'.
 
         Returns:
             str: The appropriate image URI based on the given parameters.
 
         """
-        return fw_utils.create_image_uri(
+        return create_image_uri(
             region_name,
-            "-".join([self.__framework_name__, "serving"]),
+            self.__framework_name__,
```

Comment:
Can you clarify why tensorflow-serving isn't used here?

Reply:
this class is used only for the legacy TF images - those images supported both training and serving, and were simply named "sagemaker-tensorflow." The sagemaker-tensorflow-serving images are handled by the `sagemaker.tensorflow.serving.Model` class.